frame/support/procedural/src/pallet/parse/config.rs

```diff
@@ -52,6 +52,11 @@ pub struct ConfigDef {
 	pub where_clause: Option<syn::WhereClause>,
 	/// The span of the pallet::config attribute.
 	pub attr_span: proc_macro2::Span,
+	/// Whether a default sub-trait should be generated.
+	///
+	/// No, if `None`.
+	/// Yes, if `Some(_)`, with the inner items that should be included.
+	pub default_sub_trait: Option<Vec<syn::TraitItem>>,
```

Comment:
Wouldn't an empty vec already represent no default subtrait? It doesn't make any sense to create a default trait with no items otherwise...

Reply:
I agree, this was @kianenigma's so thought he might have a reason, will rework to empty vec = no